### PR TITLE
Added member-settings on option-hold hover  & view new member contributions for super-admin

### PR DESCRIPTION
**Pull request by DashDeipayan**
### What is the change?

- On option hover for super admin, the settings button will be visible.
- On option hold and click on new members for super-admin, can see their contributions.

### Is it bug?
No

- Steps to repro
- - Enable super-admin options
- - hold option and hover
- Expected
- Actual

### \*Dev Tested?

- [x] Yes
- [ ] No

#### Platforms

- [x] Web

#### Browsers

- [x] Chrome
- [x] Safari
- [ ] Firefox

### Before / After Change Screenshots

> For visual or interaction changes. Can be video / screenshot.

https://user-images.githubusercontent.com/34045191/148844720-29836df8-6d6b-4911-811a-52db807b3f58.mov

https://user-images.githubusercontent.com/34045191/148844736-3b8d5463-4918-4ec7-8a06-ec90f8330060.mov

### CODE REVIEW

File: src/components/home/index.js [hunk 1]
```diff
@@ -19,9 +20,22 @@ const Home = () => {
...
+      role="button"
+      tabIndex={0}
+      onKeyDown={(e) => {
+        if (e.key === 'Alt') {
+          setOptionKey(true);
+        }
+      }}
+      onKeyUp={() => {
+        setOptionKey(false);
+      }}
```
Comment (sumitd94) [inline src/components/home/index.js:37]:
```suggestion
      onKeyUp={() => setOptionKey(false)}
```

Comment (sumitd94) [reply src/components/home/index.js:37]: NIT

File: src/components/member-card/index.js [hunk 2]
```diff
@@ -7,7 +7,7 @@ import PropTypes from 'prop-types';
 import ShowSkills from '@components/member-card/show-skills';
 import SuperUserOptions from '@components/member-card/super-user-options';
 
-const Card = ({ developerInfo }) => {
+const Card = ({ developerInfo, optionKey }) => {
```
Comment (sumitd94) [inline src/components/member-card/index.js:10]: I do not see `optionKey` added in the props list
https://github.com/Real-Dev-Squad/website-members/pull/303/files#diff-cf115ea7b3dbc387c34873a734a99c2166b7fb70acd94ccb24125d9d949e431dR85

File: src/components/member-card/super-user-options.js [hunk 3]
```diff
@@ -3,7 +3,7 @@ import classNames from '@components/member-card/card.module.scss';
 import PropTypes from 'prop-types';
 import { userContext } from '../../store/user/user-context';
 
-const SuperUserOptions = ({ username }) => {
+const SuperUserOptions = ({ username, showSettings }) => {
```
Comment (sumitd94) [inline src/components/member-card/super-user-options.js:6]: Add to the props list as well, please

File: src/components/member-card/super-user-options.js [hunk 4]
```diff
@@ -19,7 +19,11 @@ const SuperUserOptions = ({ username }) => {
 
   const showSuperUserOptions = () => {
     return (
-      <div className={classNames.settingsContainer}>
+      <div
+        className={`${classNames.settingsContainer} ${
+          !showSettings ? classNames.hidden : ''
```
Comment (sumitd94) [inline src/components/member-card/super-user-options.js:24]:
```suggestion
          !showSettings && classNames.hidden 
```

File: src/components/members/index.js [hunk 5]
```diff
@@ -4,7 +4,7 @@ import { membersContext } from '@store/members/members-context';
 
 import styles from '@components/members/members.module.scss';
 
-const Members = () => {
+const Members = ({ optionKey }) => {
```
Comment (sumitd94) [inline src/components/members/index.js:7]: can you please define propTypes for this component?

File: src/components/new-members/index.js [hunk 6]
```diff
@@ -1,17 +1,45 @@
...
+      }}
+      as={`/${username}`}
+      key={username}
+    >
+      {RenderNewMemberCard(newMember, optionKey)}
+    </Link>
+  ) : (
+    RenderNewMemberCard(newMember, optionKey)
+  );
+};
```
Comment (sumitd94) [inline src/components/new-members/index.js:40]: Are we creating a profiles page for new members? 🤔  If yes then it should be part of different PR and not this.
cc - @ankushdharkar

File: src/components/member-card/index.js [hunk 7]
```diff
@@ -27,8 +34,17 @@ const Card = ({ developerInfo }) => {
...
   return (
-    <div>
-      {dev && <SuperUserOptions username={username} />}
+    <div
+      role="button"
+      tabIndex={0}
+      onMouseEnter={handleSettingsButton}
+      onMouseLeave={() => {
+        setShowSettings(false);
+      }}
```
Comment (harshith-venkatesh) [inline src/components/member-card/index.js:43]:
```suggestion
      onMouseLeave={() => 
        setShowSettings(false)
      }
```

**Comment by harshith-venkatesh** [comment]
Closing the issue as the changes are already merged to main 
CC : @sumitd94 @ankushdharkar
